node/package.json

```diff
@@ -31,6 +31,7 @@
     "binary-search": "",
     "chai": "",
     "child-process-promise": "",
```

Comment:
It's a shame to have two child-process libraries.  Is there any way we can consolidate to one?  If not, let's ship this.